server/memphis_handlers_stations.go

```diff
@@ -507,6 +528,15 @@ func (sh StationsHandler) CreateStation(c *gin.Context) {
 		return
 	}
 
+	if len(body.Tags) > 0 {
+		err = AddTagsToEntity(body.Tags, "station", newStation.Name)
```

Comment:
you can send the station ID instead and not query the DB again inside this function